db/migrate/20150120095012_create_categories.rb

```diff
@@ -2,6 +2,7 @@ class CreateCategories < ActiveRecord::Migration
   def change
     create_table :categories do |t|
       t.string :name
+      t.string :link
```

Comment:
the category (it turns out) can be called things like 'Health & Fitness' but of course the link is different, ie health_fitness. So it this table needs 2 fields for each category